scripts/cheats.lua

```diff
@@ -157,8 +157,46 @@ function HandleCheats(str)
     PlayMusic("music/I'm a Medieval Man" .. wargus.music_extension)
     AddMessage("enabled cheat")
 
+  elseif (IsDebugEnabled) then
+    if (str == "fast debug") then
+      for i = 0,PlayerMax - 1 do
+        for j = 1,table.getn(resources) do
+        SetSpeedResourcesHarvest(i, resources[j], 1000)
+        SetSpeedResourcesReturn(i, resources[j], 1000)
+        end
+        SetSpeedBuild(i, 1000)
+        SetSpeedTrain(i, 1000)
+        SetSpeedUpgrade(i, 1000)
+        SetSpeedResearch(i, 1000)
+      end
+      AddMessage("FAST DEBUG SPEED")
+
+    elseif (str == "normal debug") then
+      for i = 0,PlayerMax - 1 do
+        for j = 1,table.getn(resources) do
+        SetSpeedResourcesHarvest(i, resources[j], 100)
+        SetSpeedResourcesReturn(i, resources[j], 100)
+        end
+        SetSpeedBuild(i, 100)
+        SetSpeedTrain(i, 100)
+        SetSpeedUpgrade(i, 100)
+        SetSpeedResearch(i, 100)
+      end
+      AddMessage("NORMAL DEBUG SPEED")
+
+    elseif (string.find(str, ".lua")) then
+      AddMessage("Reloading " .. str)
+      Load("scripts/" .. str)
+
+    elseif (string.find(str, "eval") == 1) then
+      local code = str:gsub("^eval%s", "")
+      AddMessage("Running: " .. code)
+      local result = loadstring("return " .. code)
+      result = result()
+      AddMessage(" => " .. tostring(result))
```

Comment:
I don't have this in the other version either, but it might be a good idea to also just `print` the evaled code and result, so it goes to the console. Otherwise there's no history.

Reply:
I took it from here
https://github.com/Wargus/war1gus/blob/1875d87b8709331553dd00695503cb7588fe41dd/scripts/cheats.lua#L204

Reply:
Ah, sorry, I misunderstood you.

Reply:
Done.